### PR TITLE
Add pr size label, `tox` to GitHub actions

### PR DESCRIPTION
**Pull request by rnetser**
To be merged after https://github.com/opendatahub-io/opendatahub-tests/pull/24

### CODE REVIEW

File: .github/workflows/comment-pr.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: 'Add Labels on PR Events'
+
+on:
+  issue_comment:
+    types: [created]
```
Comment (rnetser) [inline .github/workflows/comment-pr.yml:5]: test comment

Comment (rnetser) [inline .github/workflows/comment-pr.yml:5]: comment

File: .github/workflows/comment-pr.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: PR Comment Labeler
+
+on:
+  pull_request:
+    types: [opened, edited]
```
Comment (rnetser) [inline .github/workflows/comment-pr.yml:5]: xxx

File: .github/workflows/comment-pr.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+name: PR Comment Labeler
+
+on:
+  pull_request:
```
Comment (rnetser) [inline .github/workflows/comment-pr.yml:4]: zzzz

**Comment by rnetser** [comment]
test

**Review by adolfo-ab — APPROVED** [review]

**Pull request closed without merging**